src/navigation/SubTabs/SubTabs.styles.tsx

```diff
@@ -0,0 +1,26 @@
+import styled, { css } from 'styled-components/native'
+import { Text, TouchableOpacity } from 'react-native'
+import { TabButtonProps, TabTextProps } from './SubTabs.types'
+
+export const StyledTabButton = styled(TouchableOpacity)`
```

Comment:
```suggestion
export const StyledTabButton = styled.TouchableOpacity`
```
Tu peux aussi supprimer l'import de TouchableOpacity